tests/loggers/test_mlflow_logger.py

```diff
@@ -293,3 +295,48 @@ def test_mlflow_logging_works(tmp_path, device):
 
     expected_params_list = ['num_cpus_per_node', 'node_name', 'num_nodes', 'rank_zero_seed']
     assert set(expected_params_list) == set(actual_params_list)
+
+
+@device('cpu')
+def test_mlflow_log_image_works(tmp_path, device):
+
+    class ImageLogger(Callback):
+
+        def before_forward(self, state: State, logger: Logger):
+            inputs = state.batch_get_item(key=0)
+            images = inputs.data.cpu().numpy()
+            logger.log_images(images, step=state.timestamp.batch.value, use_table=True)
```

Comment:
```suggestion
            logger.log_images(images, step=state.timestamp.batch.value, use_table=True)
```
shouldn't need `use_table=True` here because its default true? since that arg is only for wandb, would be good to have the test not specify it

Reply:
done